.ruby-version

```diff
@@ -0,0 +1 @@
+2.3.1
```

Comment:
This version *works*, but I'm not sure it's the right version for this project. (Among other things, Circle downloads a whole new binary installation of Ruby for the build. Ick!)